compiler/modules/front.syntax/parseStmt.bal

```diff
@@ -164,24 +164,28 @@ function finishIdentifierStmt(Tokenizer tok, string identifier, Position pos, Po
     }
     else if cur == ":" {
         check tok.advance();
-        return finishOptQualIdentifierStmt(tok, identifier, check tok.expectIdentifier(), pos, startPos);
+        Position newNamePos = tok.currentStartPos();
```

Comment:
Why not `namePos`? The string variable is called `name`.

Reply:
fixed